obs-websocket.dir/Release/obs-websocket.log

```diff
@@ -0,0 +1,21 @@
+﻿  WSEvents.cpp
```

Comment:
That file slipped in!

Reply:
Add "/obs-websocket.dir/" to .gitignore

Reply:
Don't forget to remove the actual file! I can't merge these unwanted files.